src/move/econia/sources/registry.move

```diff
@@ -2481,7 +2481,7 @@ module econia::registry {
         assert!(get_market_id_base_coin<BC, QC>(
                     lot_size_2, tick_size_2, min_size_2) == option::none(), 0);
         // Assert events.
-        let market_registration_events = event::emitted_events(
+        let market_registration_events = event::emitted_events_by_handle(
```

Comment:
These changes seem unrelated?

Reply:
Yes, but the tests in the PR can't run without fixing this due to upstream changes forced by Aptos Labs